Services/CO.CDP.DataSharing.WebApi/PdfGenerator.cs

```diff
@@ -130,54 +132,54 @@ private void AddConnectedPersonInformationSection(ColumnDescriptor col, List<Con
-                    col.Item().Element(container => AddTwoColumnRow(container, "Nationality", person.Nationality));
+                    col.Item().Element(container => AddTwoColumnRow(container, StaticTextResource.PdfGenerator_ConnectedPerson_Nationality, person.Nationality));
 
                 if (person.DateOfBirth != null)
-                    col.Item().Element(container => AddTwoColumnRow(container, "Date of birth", person.DateOfBirth?.ToString("dd MMMM yyyy")));
+                    col.Item().Element(container => AddTwoColumnRow(container, StaticTextResource.PdfGenerator_ConnectedPerson_DateOfBirth, person.DateOfBirth?.ToString("dd MMMM yyyy")));
 
                 if (!string.IsNullOrEmpty(person.CompanyHouseNumber))
-                    col.Item().Element(container => AddTwoColumnRow(container, "Company House Number", person.CompanyHouseNumber));
+                    col.Item().Element(container => AddTwoColumnRow(container, StaticTextResource.PdfGenerator_ConnectedPerson_CompanyHouseNumber, person.CompanyHouseNumber));
 
                 if (!string.IsNullOrEmpty(person.PersonType.ToString()))
                 {
                     if (!string.IsNullOrEmpty(person.Organisation?.Name))
                     {
-                        col.Item().Element(container => AddTwoColumnRow(container, "Type", GetFriendlyEntityTypeText(person.EntityType)));
+                        col.Item().Element(container => AddTwoColumnRow(container, StaticTextResource.PdfGenerator_ConnectedPerson_OrganisationEntityType, GetFriendlyEntityTypeText(person.EntityType)));
                     }
                     else
                     {
-                        col.Item().Element(container => AddTwoColumnRow(container, "Type", GetFriendlyPersonTypeText(person.PersonType)));
+                        col.Item().Element(container => AddTwoColumnRow(container, StaticTextResource.PdfGenerator_ConnectedPersonType, GetFriendlyPersonTypeText(person.PersonType)));
                     }
                 }
 
                 if (!string.IsNullOrEmpty(person.Organisation?.Name))
                 {
-                    col.Item().Element(container => AddTwoColumnRow(container, "Category", GetFriendlyOrganisationCategoryText(person.ConnectedOrganisationCategoryType)));
+                    col.Item().Element(container => AddTwoColumnRow(container, StaticTextResource.PdfGenerator_ConnectedPerson_OrganisationCategory, GetFriendlyOrganisationCategoryText(person.ConnectedOrganisationCategoryType)));
                 }
                 else if (!string.IsNullOrEmpty(person.ToString()))
                 {
-                    col.Item().Element(container => AddTwoColumnRow(container, "Category", GetFriendlyCategoryText(person.Category)));
+                    col.Item().Element(container => AddTwoColumnRow(container, StaticTextResource.PdfGenerator_ConnectedPersonCategory, GetFriendlyCategoryText(person.Category)));
```

Comment:
This should be `PdfGenerator_ConnectedPerson_Type` and `PdfGenerator_ConnectedPerson_Category` to fit the pattern